isis/src/qisis/apps/ipce/IpceMainWindow.cpp

```diff
@@ -226,10 +226,10 @@ namespace Isis {
 
 
   /**
-   * @description This slot is connected from Directory::viewClosed(QWidget *) signal.  It will 
+   * @description This slot is connected from Directory::viewClosed(QWidget *) signal.  It will
    * close the given view and delete the view. This was written to handle
-   * 
-   * @param view QWidget* 
+   *
+   * @param view QWidget*
```

Comment:
What is this parameter used for? It's not used in the method so why is it here?

Reply:
It is used in the method; there's a line that closes the view with view->close().